test/integration/features/story_registry.feature

```diff
@@ -11,7 +11,7 @@ Feature: Local image to image-registry to deployment
         When starting CRC with default bundle and hypervisor "<vm-driver>" succeeds
         Then stdout should contain "CodeReady Containers instance is running"
         And executing "eval $(crc oc-env)" succeeds
-        When with up to "4" retries with wait period of "2m" command "crc status" output should contain "Running (v4.x)"
+        When with up to "4" retries with wait period of "2m" command "crc status" output should contain "Running (v4"
```

Comment:
@cfergeau you put `.` in every other place and forgot here for `v4`  :)

Reply:
Ah thanks, I remember thinking about consistency when doing these changes, but I did not double-check they actually were the same ;)